src/ahbicht/validation/validation.py

```diff
@@ -132,13 +164,14 @@ async def validate_segment(
         ValidationResultInContext(discriminator=segment.discriminator, validation_result=segment_validation)
     ]
 
-    tasks = []
-
-    # validation of this segments data elements
-    for data_element in segment.data_elements:
-        tasks.append(validate_data_element(data_element, segment_validation.requirement_validation))
-
-    validation_results_in_context_data_elements = await asyncio.gather(*tasks)
+    # validation of this segments data elements if segment is not forbidden
```

Comment:
```suggestion
    # validate this segments' data elements only if segment is not forbidden
```
haha! wieder mal ein genitiv s